src/os/ui/im/action/filteractionimporter.js

```diff
@@ -64,6 +64,17 @@ os.ui.im.action.FilterActionImporter.prototype.getFilterTooltip = function(entry
 };
 
 
+/**
+ * @param {os.layer.Vector} vectorLayer
+ * @return {Array}
+ * @override
+ * @suppress {checkTypes}
```

Comment:
Given the previous suggestion to update your convenience function, this should be able to just use `@inheritDoc` with no additional documentation. We like to avoid suppressing `checkTypes` because it's typically avoiding legitimate compiler errors.